servers/lib/server/handlers/invitetoteambycsvAnalyze.coffee

```diff
@@ -0,0 +1,101 @@
+os = require 'os'
+fs = require 'fs'
+hat = require 'hat'
+path = require 'path'
+Busboy = require 'busboy'
+parser = require 'csv-parse'
+{ generateFakeClient } = require './../client'
+
+module.exports.parserOpts = csvParseOpts =
+  delimiter: ','
+  columns: ['email', 'firstName', 'lastName', 'role']
+  relax: no # Preserve quotes inside unquoted field.
+  relax_column_count: yes
+  skip_empty_lines: true
+  trim: true # remove whitespaces
+  auto_parse: no # convert integers to numbers etc
+
+module.exports.handler = (req, rres) ->
```

Comment:
https://github.com/koding/koding/pull/8676/files#r73656920 applies to here, we can reuse whole code again, just pass a "processor"  to handler function and let  rest stay same